tests/unit/test_manage_documentation_versions.py

```
@@ -0,0 +1,609 @@
+        result = await version_manager.compare_versions(version1_id, version2_id)
+
+        # Assert
+        assert isinstance(result, VersionComparisonResult)
+        assert result.version1_id == version1_id
+        assert result.version2_id == version2_id
+        assert result.content_diff.added_lines > 0
+        assert result.content_diff.modified_lines >= 0
+        assert len(result.metadata_changes) > 0
+
+    async def test_rollback_to_version_success(
+        self, version_manager, mock_supabase_client_extended, mock_support_doc_service
```

Pylint (3.3.7)

[warning] Unused argument 'mock_support_doc_service'

(W0613)